app.rb

```diff
@@ -0,0 +1,78 @@
+require_relative 'person'
+require_relative 'book'
+require_relative 'classroom'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+
+class App
+  attr_accessor :people, :books, :rentals
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
+  end
+
+  def list_people
+    @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+  end
+
```

Comment:
- [ ] When I list books and people different data is listed also. Kindly remove the items. We only need to list the basic information about the books and people, not the object memory.

  ![laos](https://user-images.githubusercontent.com/54848363/231782862-9863b7b6-2b7f-439f-89fe-0d9166f983f0.jpg)
  ![iisos](https://user-images.githubusercontent.com/54848363/231782866-516a7efa-1645-489e-ba2e-7dfb90db0cee.jpg)